### DISCUSSION TITLE
Copilot refund after getting student benefits

### DISCUSSION THREAD

**Discussion by MuteKnowLESS** [community/community#39061] [GitHub Education] [answered]
A few months ago, I started trialing GitHub Copilot. Subsequently, before my Copilot trial ended, I learnt that students are eligible for a free Github Copilot subscription. After receiving the student benefits, I forgot to cancel my paid Copilot subscription and only realised after receiving a $83 bill that the subscription does not cancel automatically. I would like to request a refund for said payment. I opened a GitHub support ticket [#1875774] a few days ago and have gotten no response. I would like to resolve this quickly because $80 is still a considerable amount of money and I don’t want to get stuck in that fine print 14 day kind of policy many products have.

**Comment by MuteKnowLESS** [comment, marked as answer]
Issue resolved with GitHub internally

**Reply by Biantuadi** [reply]
how did they refund you?

**Reply by nduc86686** [reply]
Please,how did you solve it?

**Reply by MuteKnowLESS** [reply]
I linked my original support ticket which was entered under the general scope to another support ticket under the student benefits scope. They replied within a few hours and sent the money back to my card.

**Reply by Biantuadi** [reply]
How many days take u to see money in your count?

**Reply by quangtuan28200** [reply]
i subscribed to github copilot 60 days free trial package yesterday, but today i got $100 deducted. I thought within 60 days, if I cancel it, there will be no charge? how to solve it

**Reply by sisansahu1** [reply]
did they refund?
